goreleaser.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+project_name: gh-ghn
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - freebsd
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+      - arm
```

Comment:
```suggestion
      - arm
```

Reply:
(this is only needed for 32bit -- practically no one is using 32bit processors e.g. raspberry pi 1! for such tools so we can remove this too)